Task/Taskfile.yml

```diff
@@ -4,6 +4,107 @@
 version: "3"
 
 tasks:
+  lint:
+    preconditions:
+      - sh: docker pull seiso/goat:latest || echo "Failed to pull the latest goat, continuing anyway..."
+    # Adjustments are passed as vars, not envs, to avoid local workarounds
+    vars:
+      INPUT_AUTO_FIX: '{{.INPUT_AUTO_FIX | default ""}}'
+      INPUT_DISABLE_MYPY: '{{.INPUT_DISABLE_MYPY | default ""}}'
+      INPUT_EXCLUDE: '{{.INPUT_EXCLUDE | default ""}}'
+      INPUT_LOG_LEVEL: '{{.INPUT_LOG_LEVEL | default ""}}'
+    cmds:
+      - |
+        docker run --env INPUT_AUTO_FIX="{{.INPUT_AUTO_FIX}}" \
+                   --env INPUT_DISABLE_MYPY="{{.INPUT_DISABLE_MYPY}}" \
+                   --env INPUT_EXCLUDE="{{.INPUT_EXCLUDE}}" \
+                   --env INPUT_LOG_LEVEL="{{.INPUT_LOG_LEVEL}}" \
+                   --rm -v "{{.ROOT_DIR}}:/goat" seiso/goat:latest
+
+  build:
+    vars:
+      VERSION: '{{.VERSION | default "UNKNOWN"}}'
```

Comment:
This is new; uses the `default` function and will drop to "UNKNOWN" but otherwise takes the provided `VERSION` and uses it.  Downstream (`bash/Taskfile.yml` and `python/`) should be passing a version with logic specific to bash/python.